README.md

```diff
@@ -38,7 +31,7 @@ and decompressing it, the image can be written to an SD card.
 Build the gadget snap with snapcraft:
 
 ```bash
-cd turtlebot3c-gadget
+cd pi-gadget
```

Comment:
The gadget is called turtlebot3c-gadget later in the PR. Why using pi-gadget here?